tests/integration/src/test/java/org/apache/pulsar/tests/integration/loadbalance/ExtensibleLoadManagerTest.java

```diff
@@ -346,7 +350,8 @@ public void testIsolationPolicy() throws PulsarAdminException {
             fail();
         } catch (Exception ex) {
             log.error("Failed to lookup topic: ", ex);
-            assertTrue(ex.getMessage().contains("Failed to look up a broker"));
+            assertTrue(ex.getMessage().contains("Failed to look up a broker")
```

Comment:
Perhaps you can try to use assertj to assert contains so that we know the message instead of "expect true, actual false".

Reply:
Makes sense. I didn't realize that we already have AssertJ as a dependency since it's rarely used in the Pulsar code base.

Reply:
@tisonkun I switched to use AssertJ. However, there was a need to upgrade the AssertJ version since `.containsAnyOf` wasn't available in the version that was used. That is available since AssertJ 3.21.0, we were using 3.18.1 . I upgraded to latest AssertJ version which is 3.24.2 .

Reply:
Thank you! I'm not faimilar with this code so I cannot give an approval without more details. But @BewareMyPower already approves so it can be fine, with his endorsement :D

Reply:
Yeah, it LGTM.